bazel/repository_locations.bzl

```diff
@@ -301,6 +301,33 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         release_date = "2020-08-25",
         cpe = "N/A",
     ),
+    skywalking_data_collect_protocol = dict(
+        project_name = "skywalking-data-collect-protocol",
+        project_desc = "Data Collect Protocols of Apache SkyWalking",
+        project_url = "https://github.com/apache/skywalking-data-collect-protocol",
+        name = "skywalking_data_collect_protocol",
+        sha256 = "fa9ac679624217f30b6e8d5c450365386c610e2d08188a20f0340c3b14401833",
+        urls = ["https://github.com/apache/skywalking-data-collect-protocol/archive/v8.3.0.zip"],
+        strip_prefix = "skywalking-data-collect-protocol-8.3.0",
+        version = "8.3.0",
+        use_category = ["observability_ext"],
+        extensions = ["envoy.tracers.skywalking"],
+        release_date = "2020-11-20",
+        cpe = "N/A",
+    ),
+    com_github_skyapm_cpp2sky = dict(
```

Comment:
This is a new dependency, can you please provide an evaluation against the criteria in https://github.com/envoyproxy/envoy/blob/master/DEPENDENCY_POLICY.md#new-external-dependencies? CC @moderation

Reply:
@htuch Output from [scorecard](https://github.com/ossf/scorecard) is not great (sorted for Pass / Fail). Average score is -0.142857142857143.
```
scorecard --repo=https://github.com/SkyAPM/cpp2sky

RESULTS
-------
Active: Pass 10
CI-Tests: Pass 9
Code-Review: Pass 10
Frozen-Deps: Pass 10
Pull-Requests: Pass 9
Security-Policy: Pass 10

Branch-Protection: Fail 10
CII-Best-Practices: Fail 10
Contributors: Fail 10
Fuzzing: Fail 10
Packaging: Fail 10
SAST: Fail 10
Signed-Releases: Fail 0
Signed-Tags: Fail 0
```

Reply:
@moderation Could you show me the average score of scorecard to be valid? I'm a maintainer of this dependency so can do efforts to improve this score.

Reply:
@moderation I cheched `dd-opentracing-cpp` and it results in `-2.5`. So I think it is enough managed.

```
./scorecard --repo=https://github.com/DataDog/dd-opentracing-cpp

RESULTS
-------
CI-Tests: Pass 10
Code-Review: Pass 10
Pull-Requests: Pass 10
Signed-Releases: Pass 10

Active: Fail 10
Branch-Protection: Fail 10
CII-Best-Practices: Fail 10
Contributors: Fail 10
Frozen-Deps: Fail 5
Fuzzing: Fail 10
Packaging: Fail 0
SAST: Fail 10
Security-Policy: Fail 10
Signed-Tags: Fail 0
```

Reply:
@Shikugawa you might want to configure branch protection at least, that's easy to do in GH.

Reply:
@htuch Branch protection has already hired on cpp2sky. It is only protection to push main branch directly. Is it not enough?